app/controllers/books_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class BooksController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    books = Book.all.page(1)
+    render json: books
```

Comment:
no need to assign a a value to a variable before rendering it, you can directly do `render json: Book.all`